src/gel-grid.scss

```diff
@@ -3,8 +3,8 @@
 //\*------------------------------------*/
 
 // Include general dependancies
-@import '../node_modules/gel-sass-tools/sass-tools';
```

Comment:
Would like to point out that this forces people to have to use node, they should be setting gel-sass-tools and sass-mq in their includes path when using sass as I have done in your script